ggshield/cmd/utils/common_options.py

```diff
@@ -187,13 +188,52 @@ def decorator(cmd: AnyFunction) -> AnyFunction:
     return decorator
 
 
+def _set_json_output_format(
+    ctx: click.Context, param: click.Parameter, value: Optional[bool]
+) -> Optional[bool]:
+    if value:
+        ctx_obj = ContextObj.get(ctx)
+        ctx_obj.output_format = OutputFormat.JSON
+    return value
+
+
 json_option = click.option(
     "--json",
     "json_output",
     is_flag=True,
     default=None,
     help="Use JSON output.",
-    callback=create_ctx_callback("use_json"),
+    callback=_set_json_output_format,
+)
+
+
+def _set_output_format(
```

Comment:
So, are we planning to deprecate `--json` over time, or will we keep the command ?